 
Data Hora: 14/03/2018 9:22:16
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.DbContextTransaction.EnsureOpenConnection()
   em System.Data.Entity.DbContextTransaction..ctor(EntityConnection connection)
   em System.Data.Entity.Database.BeginTransaction()
   em Faturamento.ApplicationService.Service.DocumentosaidaService.Salva(DocumentosaidaView entidadePostado, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs:linha 1018
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 342

--------------------------------------------
 
Data Hora: 14/03/2018 9:22:18
Rotina:    System.Data.Common.DbConnection get_Connection()
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: ERRO_InvalidOperationException
Título:    The ObjectContext instance has been disposed and can no longer be used for operations that require a connection.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.Objects.ObjectContext.get_Connection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.ParametroService.Get(String pParametro, Nullable`1 pIdFilial) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\ParametroService.cs:linha 70
   em Faturamento.ApplicationService.Helpers.FaturamentoHelper.isComissaoPorRegra() na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Helpers\FaturamentoHelper.cs:linha 29
   em Faturamento.ApplicationService.Service.DocumentosaidaService.Salva(DocumentosaidaView entidadePostado, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs:linha 1039
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 342

--------------------------------------------
 
Data Hora: 14/03/2018 9:22:55
Rotina:    Void InitializeContext()
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: ERRO_InvalidOperationException
Título:    The operation cannot be completed because the DbContext has been disposed.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Internal.LazyInternalContext.InitializeContext()
   em System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   em System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   em System.Data.Entity.Internal.Linq.InternalSet`1.get_InternalContext()
   em System.Data.Entity.Infrastructure.DbQuery`1.System.Linq.IQueryable.get_Provider()
   em System.Linq.Queryable.Where[TSource](IQueryable`1 source, Expression`1 predicate)
   em PainelControle.ApplicationService.Service.ParametroService.Get(String pParametro, Nullable`1 pIdFilial) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\ParametroService.cs:linha 70
   em PainelControle.ApplicationService.Service.ParametroService.Get[T](String pParametro, Nullable`1 pIdFilial) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\ParametroService.cs:linha 102
   em Financeiro.ApplicationService.Helpers.ContasareceberHelper.GetModeloJurosSelecionado(Nullable`1 pIdFilial) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Financeiro\Financeiro.ApplicationService\Helpers\ContasareceberHelper.cs:linha 875
   em Financeiro.ApplicationService.Service.ContasareceberService.Get(Int32 id, Nullable`1 pDataQuitacao) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Financeiro\Financeiro.ApplicationService\Service\ContasareceberService.cs:linha 276
   em Financeiro.ApplicationService.Service.ContasareceberService.BaixaTituloNormal(Conta pCaixa, Contasareceber tituloPostado, DbContextTransaction pTransaction, Boolean inclusao, Int64 lote) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Financeiro\Financeiro.ApplicationService\Service\ContasareceberService.cs:linha 1525
   em Financeiro.ApplicationService.Service.ContasareceberService.BaixaTitulo(Contasareceber titulo, TipoBaixa tipoBaixa, Boolean inclusao, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Financeiro\Financeiro.ApplicationService\Service\ContasareceberService.cs:linha 1413
   em Financeiro.ApplicationService.Service.ContasareceberService.Salva(Contasareceber tituloPostado, Nullable`1 pNumeroTitulo, Boolean inclusaoAutomatica, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Financeiro\Financeiro.ApplicationService\Service\ContasareceberService.cs:linha 479
   em Faturamento.ApplicationService.Service.DocumentosaidaService.GeraTitulosAReceber(Documentosaida documento, IList`1 parcelas, TipoAcao acao, Conta pConta, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs:linha 2407
   em Faturamento.ApplicationService.Service.DocumentosaidaService.GeraFinanceiro(Documentosaida documento, IList`1 parcelas, TipoAcao acao, Conta pContaCaixa, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs:linha 2335
   em Faturamento.ApplicationService.Service.DocumentosaidaService.Salva(DocumentosaidaView entidadePostado, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs:linha 1100
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 342

--------------------------------------------
 
Data Hora: 14/03/2018 9:23:25
Rotina:    System.Data.Common.DbDataReader ExecuteStoreCommands(System.Data.Entity.Core.EntityClient.EntityCommand, System.Data.CommandBehavior)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: ERRO_EntityCommandExecutionException
Título:    Já existe um DataReader aberto associado a este Command que deve ser fechado primeiro.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   em System.Data.Entity.Core.Objects.Internal.ObjectQueryExecutionPlan.Execute[TResultType](ObjectContext context, ObjectParameterCollection parameterValues)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__a()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.ParametroService.Get(String pParametro, Nullable`1 pIdFilial) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\ParametroService.cs:linha 70
   em PainelControle.ApplicationService.Service.ParametroService.Get[T](String pParametro, Nullable`1 pIdFilial) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\ParametroService.cs:linha 102
   em Financeiro.ApplicationService.Helpers.ContasareceberHelper.GetModeloJurosSelecionado(Nullable`1 pIdFilial) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Financeiro\Financeiro.ApplicationService\Helpers\ContasareceberHelper.cs:linha 875
   em Financeiro.ApplicationService.Service.ContasareceberService.Get(Int32 id, Nullable`1 pDataQuitacao) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Financeiro\Financeiro.ApplicationService\Service\ContasareceberService.cs:linha 276
   em Financeiro.ApplicationService.Service.ContasareceberService.BaixaTituloNormal(Conta pCaixa, Contasareceber tituloPostado, DbContextTransaction pTransaction, Boolean inclusao, Int64 lote) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Financeiro\Financeiro.ApplicationService\Service\ContasareceberService.cs:linha 1525
   em Financeiro.ApplicationService.Service.ContasareceberService.BaixaTitulo(Contasareceber titulo, TipoBaixa tipoBaixa, Boolean inclusao, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Financeiro\Financeiro.ApplicationService\Service\ContasareceberService.cs:linha 1413
   em Financeiro.ApplicationService.Service.ContasareceberService.Salva(Contasareceber tituloPostado, Nullable`1 pNumeroTitulo, Boolean inclusaoAutomatica, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Financeiro\Financeiro.ApplicationService\Service\ContasareceberService.cs:linha 479
   em Faturamento.ApplicationService.Service.DocumentosaidaService.GeraTitulosAReceber(Documentosaida documento, IList`1 parcelas, TipoAcao acao, Conta pConta, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs:linha 2407
   em Faturamento.ApplicationService.Service.DocumentosaidaService.GeraFinanceiro(Documentosaida documento, IList`1 parcelas, TipoAcao acao, Conta pContaCaixa, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs:linha 2335
   em Faturamento.ApplicationService.Service.DocumentosaidaService.Salva(DocumentosaidaView entidadePostado, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs:linha 1100
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 342

--------------------------------------------
 
Data Hora: 14/03/2018 9:23:31
Rotina:    Void InitializeContext()
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: ERRO_InvalidOperationException
Título:    The operation cannot be completed because the DbContext has been disposed.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Internal.LazyInternalContext.InitializeContext()
   em System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   em System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   em System.Data.Entity.Internal.Linq.InternalSet`1.get_InternalContext()
   em System.Data.Entity.Infrastructure.DbQuery`1.System.Linq.IQueryable.get_Provider()
   em System.Linq.Queryable.Where[TSource](IQueryable`1 source, Expression`1 predicate)
   em PainelControle.ApplicationService.Service.ParametroService.Get(String pParametro, Nullable`1 pIdFilial) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\ParametroService.cs:linha 70
   em PainelControle.ApplicationService.Service.ParametroService.Get[T](String pParametro, Nullable`1 pIdFilial) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\ParametroService.cs:linha 102
   em Financeiro.ApplicationService.Helpers.ContasareceberHelper.GetModeloJurosSelecionado(Nullable`1 pIdFilial) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Financeiro\Financeiro.ApplicationService\Helpers\ContasareceberHelper.cs:linha 875
   em Financeiro.ApplicationService.Service.ContasareceberService.Get(Int32 id, Nullable`1 pDataQuitacao) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Financeiro\Financeiro.ApplicationService\Service\ContasareceberService.cs:linha 276
   em Financeiro.ApplicationService.Service.ContasareceberService.BaixaTituloNormal(Conta pCaixa, Contasareceber tituloPostado, DbContextTransaction pTransaction, Boolean inclusao, Int64 lote) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Financeiro\Financeiro.ApplicationService\Service\ContasareceberService.cs:linha 1525
   em Financeiro.ApplicationService.Service.ContasareceberService.BaixaTitulo(Contasareceber titulo, TipoBaixa tipoBaixa, Boolean inclusao, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Financeiro\Financeiro.ApplicationService\Service\ContasareceberService.cs:linha 1413
   em Financeiro.ApplicationService.Service.ContasareceberService.Salva(Contasareceber tituloPostado, Nullable`1 pNumeroTitulo, Boolean inclusaoAutomatica, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Financeiro\Financeiro.ApplicationService\Service\ContasareceberService.cs:linha 479
   em Faturamento.ApplicationService.Service.DocumentosaidaService.GeraTitulosAReceber(Documentosaida documento, IList`1 parcelas, TipoAcao acao, Conta pConta, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs:linha 2407
   em Faturamento.ApplicationService.Service.DocumentosaidaService.GeraFinanceiro(Documentosaida documento, IList`1 parcelas, TipoAcao acao, Conta pContaCaixa, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs:linha 2335
   em Faturamento.ApplicationService.Service.DocumentosaidaService.Salva(DocumentosaidaView entidadePostado, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs:linha 1100
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 342

--------------------------------------------
 
Data Hora: 14/03/2018 9:24:19
Rotina:    Void InitializeContext()
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: ERRO_InvalidOperationException
Título:    The operation cannot be completed because the DbContext has been disposed.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Internal.LazyInternalContext.InitializeContext()
   em System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   em System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   em System.Data.Entity.Internal.Linq.InternalSet`1.get_InternalContext()
   em System.Data.Entity.Internal.Linq.InternalSet`1.ActOnSet(Action action, EntityState newState, Object entity, String methodName)
   em System.Data.Entity.Internal.Linq.InternalSet`1.Attach(Object entity)
   em System.Data.Entity.DbSet`1.Attach(TEntity entity)
   em Financeiro.ApplicationService.Helpers.ContasareceberHelper.AnexarVirtuaisSalvar(Contasareceber entidade) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Financeiro\Financeiro.ApplicationService\Helpers\ContasareceberHelper.cs:linha 449
   em Financeiro.ApplicationService.Service.ContasareceberService.Salva(Contasareceber tituloPostado, Nullable`1 pNumeroTitulo, Boolean inclusaoAutomatica, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Financeiro\Financeiro.ApplicationService\Service\ContasareceberService.cs:linha 369
   em Faturamento.ApplicationService.Service.DocumentosaidaService.GeraTitulosAReceber(Documentosaida documento, IList`1 parcelas, TipoAcao acao, Conta pConta, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs:linha 2407
   em Faturamento.ApplicationService.Service.DocumentosaidaService.GeraFinanceiro(Documentosaida documento, IList`1 parcelas, TipoAcao acao, Conta pContaCaixa, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs:linha 2335
   em Faturamento.ApplicationService.Service.DocumentosaidaService.Salva(DocumentosaidaView entidadePostado, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs:linha 1100
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 342

--------------------------------------------
 
Data Hora: 14/03/2018 10:0:48
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/estoque/produto/dicionario/
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.ProdutoController.GetDicionario(HttpRequestMessage request) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Estoque\ProdutoController.cs:linha 47

--------------------------------------------
 
Data Hora: 14/03/2018 10:12:19
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/pedido/dicionario/
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.PedidoController.GetDicionario(HttpRequestMessage request) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\PedidovendaController.cs:linha 48

--------------------------------------------
 
Data Hora: 14/03/2018 10:19:17
Rotina:    System.Data.Common.DbDataReader ExecuteStoreCommands(System.Data.Entity.Core.EntityClient.EntityCommand, System.Data.CommandBehavior)
Rota:      http://localhost:2020/api/v1/integracao/empresa/filiais
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: ERRO_EntityCommandExecutionException
Título:    Ocorreu um erro no nível de transporte durante o recebimento de resultados do servidor. (provider: TCP Provider, error: 0 - O tempo limite do semáforo expirou.)
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   em System.Data.Entity.Core.Objects.Internal.ObjectQueryExecutionPlan.Execute[TResultType](ObjectContext context, ObjectParameterCollection parameterValues)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__a()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Controllers.EmpresaController.Filiais(HttpRequestMessage request) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Integracao\EmpresaController.cs:linha 138

--------------------------------------------
 
Data Hora: 14/03/2018 10:28:53
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 14/03/2018 10:45:31
Rotina:    Void MoveNext()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs
Linha:     114
Tipo Erro: 1
Nome Erro: TOK0004
Título:    Condição Falha
Motivo:    Erro de Sistema
Solução :  Contacte o suporte!
-----------------TRACE----------------------
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 114

--------------------------------------------
 
Data Hora: 14/03/2018 10:45:55
Rotina:    Void MoveNext()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs
Linha:     114
Tipo Erro: 1
Nome Erro: TOK0004
Título:    Condição Falha
Motivo:    Erro de Sistema
Solução :  Contacte o suporte!
-----------------TRACE----------------------
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 114

--------------------------------------------
 
Data Hora: 14/03/2018 10:46:0
Rotina:    Void MoveNext()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs
Linha:     114
Tipo Erro: 1
Nome Erro: TOK0004
Título:    Condição Falha
Motivo:    Erro de Sistema
Solução :  Contacte o suporte!
-----------------TRACE----------------------
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 114

--------------------------------------------
 
Data Hora: 14/03/2018 10:46:19
Rotina:    Void MoveNext()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs
Linha:     114
Tipo Erro: 1
Nome Erro: TOK0004
Título:    Condição Falha
Motivo:    Erro de Sistema
Solução :  Contacte o suporte!
-----------------TRACE----------------------
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 114

--------------------------------------------
 
Data Hora: 14/03/2018 11:19:46
Rotina:    DocumentosFiscais.NFE_NFCE.Modelos.RetornoTransmissaoNFe GerarXmlETransmitir(Int64)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Framework\DocumentosFiscais\NFE_NFCE\NFeTransmissaoService.cs
Linha:     406
Tipo Erro: 1
Nome Erro: Erro Não Mapeado
Título:    Erros na validação:
 O elemento 'http://www.portalfiscal.inf.br/nfe:CFOP' é inválido - O valor '0' é inválido dependendo do tipo de dados 'String' - Falha na restrição Pattern.
Motivo:    
Solução :  
-----------------TRACE----------------------
   em DocumentosFiscais.NFE_NFCE.NFeTransmissaoService.GerarXmlETransmitir(Int64 numeroLote) na D:\DESENVOLVIMENTO\Projetos\PLUS\Framework\DocumentosFiscais\NFE_NFCE\NFeTransmissaoService.cs:linha 406
   em Faturamento.ApplicationService.Service.NFeService.Transmitir(Empresa pEmpresa, Cliente pCliente, Documentosaida pDocumento, Int64 numeroLote, Boolean pOffLine, DadosTerminal dadosTerminal) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\NFeService.cs:linha 690
   em Faturamento.ApplicationService.Service.DocumentosaidaService.TransmitirDocumento(Documentosaida pDocumento, Boolean pOffline, String pJustificativa) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs:linha 2719
   em Faturamento.ApplicationService.Service.DocumentosaidaService.Salva(DocumentosaidaView entidadePostado, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs:linha 1157
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 342

--------------------------------------------
 
Data Hora: 14/03/2018 12:3:58
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/pedido/dicionario/
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.PedidoController.GetDicionario(HttpRequestMessage request) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\PedidovendaController.cs:linha 48

--------------------------------------------
 
Data Hora: 14/03/2018 14:37:30
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/pedido/dicionario/
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.PedidoController.GetDicionario(HttpRequestMessage request) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\PedidovendaController.cs:linha 48

--------------------------------------------
 
Data Hora: 14/03/2018 14:50:27
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/estoque/produto/dicionario/
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.ProdutoController.GetDicionario(HttpRequestMessage request) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Estoque\ProdutoController.cs:linha 47

--------------------------------------------
 
Data Hora: 14/03/2018 15:10:48
Rotina:    System.Data.Common.DbTransaction BeginDbTransaction(System.Data.IsolationLevel)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    An error occurred while starting a transaction on the provider connection. See the inner exception for details.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.BeginDbTransaction(IsolationLevel isolationLevel)
   em System.Data.Entity.Core.EntityClient.EntityConnection.BeginTransaction()
   em System.Data.Entity.DbContextTransaction..ctor(EntityConnection connection)
   em System.Data.Entity.Database.BeginTransaction()
   em Faturamento.ApplicationService.Service.DocumentosaidaService.Salva(DocumentosaidaView entidadePostado, DbContextTransaction pTransaction) na D:\DESENVOLVIMENTO\Projetos\PLUS\MODULOS\Faturamento\Faturamento.ApplicationService\Service\DocumentosaidaService.cs:linha 1018
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 342

--------------------------------------------
 
Data Hora: 14/03/2018 16:7:49
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/pedido/dicionario/
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.PedidoController.GetDicionario(HttpRequestMessage request) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\PedidovendaController.cs:linha 48

--------------------------------------------
 
Data Hora: 14/03/2018 16:57:13
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/fiscal/modelotributacao/dicionario/
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.ModelotributacaoController.GetDicionario(HttpRequestMessage request) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Fiscal\ModelotributacaoController.cs:linha 42

--------------------------------------------
